src/util-file.c

```diff
@@ -36,6 +36,7 @@
 #include "app-layer-parser.h"
 #include "util-validate.h"
 #include "rust.h"
+#include "stdbool.h"
```

Comment:
should not be needed

Reply:
it's also wrong, system headers should be `<stdbool.h>`. But it doesn't belong here at all.